README.md

```diff
@@ -84,6 +84,6 @@ You can find tasks on the [project board](https://github.com/solana-dev-adv/sola
 or create an issue and assign it to yourself.
 
 ### Contribution Rewards
-First 25 succesful contributers to the Solana Cookbook will get an exculsive Glasseater NFT from the [Superteam DAO](https://superteam.fun). Use this NFT to get direct access to exclusive Solana Ecosystem jobs. If you have already contirbuted, [claim your Glasseater NFT now](https://alpha.layer3.xyz/task/contribute-to-the-solana-cookbook)!
+The first 25 successful contributors to the Solana Cookbook will get an exclusive Glasseater NFT from the [Superteam DAO](https://superteam.fun). Use this NFT to get direct access to exclusive Solana Ecosystem jobs. If you have already contributed, [claim your Glasseater NFT now](https://alpha.layer3.xyz/task/contribute-to-the-solana-cookbook)!
```

Comment:
As per @jacobcreech 's comment seen here: https://github.com/solana-dev-adv/solana-cookbook/pull/198#issuecomment-1013597730 we should probably add a note about requiring code changes to claim the NFT. We are also past 25 successful contributors so this note should be either removed or updated.

Reply:
I thought this was added before. While we are past 25 contributors, I don't know if all have claimed their NFT

Reply:
Yea looks like, at the moment, the wording doesn't specify that the contributions need to be code changes. So looks like we just need to add that and this should be good. @AmaanSayyad

Reply:
Sure, will do.